app/views/layouts/application.html.erb

```diff
@@ -24,8 +24,45 @@
             </div>
           <!--jumbotron -->
       </header>
+        <div class="container"> 
+          <div class = "row">
+              <%= yield %>
+              <br>
+          </div>
+        </div>
+=======
```

Comment:
This file will cause problem. Looks like the conflict is partially resolved. If you didn't change anything inside this file, just `accept the incoming change`. Otherwise change accordingly.

Reply:
Ah, I see. Thanks alot. Will do.